src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
I'm afraid I may be misunderstanding the problem. Are you talking about hard coding a transformation matrix from D65 to D50, and another for staying within D65? Are you suggesting something like this:

```rust
pub trait RgbStandard {
    type Space: RgbSpace;
    type WhitePoint: WhitePoint;
    type TransferFn: TransferFn;
}
```

or something like this:

```rust
pub trait RgbStandard {
    type Primaries: Primaries;
    type WhitePoint: WhitePoint;
    type TransferFn: TransferFn;
}
```

or something else? Would you mind giving an example?